lib/vagrant-libvirt/config.rb

```diff
@@ -766,7 +782,11 @@ def finalize!
         # Attach mgmt network
         @mgmt_attach = true if @mgmt_attach == UNSET_VALUE
 
+        # Additional qemu commandline arguments
         @qemu_args = [] if @qemu_args == UNSET_VALUE
+
+        # Additional qemu commandline environment variables
+        @qemu_env = {} if @qemu_env == UNSET_VALUE
```

Comment:
I'm pretty sure you need to add something to handle `qemu_args` and `qemu_env` to the `def merge` function in this file otherwise if a user was to specify some settings in ~/.vagrant.d/Vagrantfile for the libvirt provider to always set some `qemu_args/qemu_env` any settings to these in the Vagrantfile of the directory would overwrite rather than merge as the user would expect.

Reply:
Added the merge update and tested locally.